Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2366,12 +2366,11 @@ extension BrowserViewController: WKUIDelegate {
         screenshotHelper.takeScreenshot(currentTab)
 
         // If the page uses window.open() or target="_blank", open the page in a new tab.
-        // TODO: This doesn't work for window.open() without user action (bug 1124942).
         let newTab: Tab
         if #available(iOS 9, *) {
-            newTab = tabManager.addTab(navigationAction.request, configuration: configuration, isPrivate: currentTab.isPrivate)
+            newTab = tabManager.addTab(navigationAction.request, configuration: configuration, afterCurrentTab: true, isPrivate: currentTab.isPrivate)
```

Comment:
How about we pass the tab to append after rather than a `Bool`? That will make the API a bit more flexible, and will also help to avoid cases where the selected tab might have changed since receiving the message, or where the message was triggered by a background tab.